schemas/pioneer-debug/deletion-request/deletion-request.1.schema.json

```diff
@@ -0,0 +1,8 @@
+{
+  "$comment": "Bug 1646151 - A deletion request in the Pioneer v2 pipeline.",
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "additionalProperties": false,
+  "properties": {},
```

Comment:
I presume this doesn't have pioneerId because it will be wrapped in a pioneer envelope?

Reply:
Yes, the pioneerId gets injected with the metadata during ingestion from the envelope.